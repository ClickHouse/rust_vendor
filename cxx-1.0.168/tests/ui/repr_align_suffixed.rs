#[cxx::bridge]
mod ffi {
    #[repr(align(2int))]
    struct StructSuffix {
        i: i32,
    }
}

fn main() {}
