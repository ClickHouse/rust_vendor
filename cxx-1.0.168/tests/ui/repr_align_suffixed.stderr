error: invalid suffix `int` for number literal
 --> tests/ui/repr_align_suffixed.rs:3:18
  |
3 |     #[repr(align(2int))]
  |                  ^^^^ invalid suffix `int`
  |
  = help: the suffix must be one of the numeric types (`u32`, `isize`, `f32`, etc.)

error[E0693]: incorrect `repr(align)` attribute format: `align` takes exactly one argument in parentheses
 --> tests/ui/repr_align_suffixed.rs:1:1
  |
1 | #[cxx::bridge]
  | ^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `cxx::bridge` (in Nightly builds, run with -Z macro-backtrace for more info)
